go.mod

```diff
@@ -3,19 +3,30 @@ module github.com/gardener/machine-controller-manager-provider-vsphere
 go 1.13
 
 require (
-	github.com/gardener/machine-spec v0.5.0
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/onsi/ginkgo v1.8.0
+	github.com/gardener/machine-controller-manager v0.0.0-20200512162209-f074b63d505f
+	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.7.1
 	github.com/pkg/errors v0.8.1
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.5
 	github.com/vmware/govmomi v0.21.1-0.20190909001527-8d286461ab92
 	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f
-	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
-	golang.org/x/sys v0.0.0-20190712062909-fae7ac547cb7 // indirect
-	google.golang.org/genproto v0.0.0-20190716160619-c506a9f90610 // indirect
-	google.golang.org/grpc v1.22.0
-	gopkg.in/inf.v0 v0.9.1 // indirect
-	k8s.io/api v0.0.0-20191114100352-16d7abae0d2a // kubernetes-1.16.3
-	sigs.k8s.io/yaml v1.1.0
+	golang.org/x/net v0.0.0-20191126235420-ef20fe5d7933
+	k8s.io/api v0.18.2 // kubernetes-1.16.3
+	k8s.io/component-base v0.18.2
+	k8s.io/klog v1.0.0
+	sigs.k8s.io/yaml v1.2.0
+)
+
+replace (
+	github.com/gardener/machine-controller-manager => github.com/prashanth26/machine-controller-manager v0.0.0-20200512162209-f074b63d505f
+	github.com/onsi/gomega => github.com/onsi/gomega v1.5.0
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.2
+	k8s.io/api => k8s.io/api v0.0.0-20190918155943-95b840bb6a1f // kubernetes-1.16.0
```

Comment:
Hi Martin,

Do you mean something like this - https://github.com/gardener/machine-controller-manager-provider-vsphere/pull/1/commits/ed78cf8a3058246ba905372bfca8fde158fcb5cb? And also why this suggestion? Is it because of the go deps handles tags with v prefix better?

Reply:
It's only for consistency with other gardener projects.

Reply:
Okay makes sense. Thanks for that. I hope these changes are fine.